azure/overnight-test-against-internal-dev-pipeline.yml

```diff
@@ -39,6 +41,32 @@ steps:
       command: 'install'
       workingDir: '$(Pipeline.Workspace)/s'
 
+  - template: "azure/components/aws-assume-role.yml@common"
+    parameters:
+     role: "auto-ops"
+     profile: "apm_ptl"    
+
+  - template: azure/components/get-aws-secrets-and-ssm-params.yml@common
+    parameters:
+     secret_file_ids:
+      - ptl/app-credentials/jwt_testing/non-prod/JWT_TESTING_PRIVATE_KEY
```

Comment:
Does the overnight pipeline need this and `TEST_PATIENT_ID`?

Reply:
Yes, TEST_PATIENT_ID is used by test_patient_create.py and test_patient_access.py scenarios